python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_resources_v2.py

```diff
@@ -520,3 +520,14 @@ def test_to_default_asset_output_events() -> None:
         "Execution Duration": FloatMetadataValue(60.0),
         "rows_affected": IntMetadataValue(100),
     }
+
+
+def test_dbt_cli_no_jinja_log_info() -> None:
+    dbt = DbtCliResource(project_dir=test_jaffle_shop_path)  # type: ignore
```

Comment:
nit: remove directive

```suggestion
    dbt = DbtCliResource(project_dir=os.fspath(test_jaffle_shop_path))
```